products/react-native.md

```diff
@@ -0,0 +1,142 @@
+---
+title: React Native
+category: framework
+tags: meta javascript-runtime
+iconSlug: react
+permalink: /react-native
+releasePolicyLink: https://reactnative.dev/versions
```

Comment:
The version list page doesn't offer any helpful information. This seems much better.

```suggestion
releasePolicyLink: https://github.com/reactwg/react-native-releases/blob/main/docs/support.md
```